docs/guide/create_a_track.md

```diff
@@ -17,9 +17,9 @@ your build settings should look like that: <br>
 <img src="../assets/create_a_track/build_settings.png" height="512">
 
 ## Apply the scene prefab
-1) In "Assets/Prefabs/" you will find a Prefab called "ScenePrefab".
+1) In "Assets/Prefabs/" you will find a Prefab called "ScenePrefab". (if not maybe you can find it in Assets/Scenes)
```

Comment:
That's right sorry, you can directly change the "Assets/Prefabs/" to "Assets/Scenes", no need for parentheses you are totally right they are now in the Scenes folder !